packages/app/src/app/pages/Dashboard/Components/Branch/BranchCard.tsx

```diff
@@ -57,7 +60,7 @@ export const BranchCard: React.FC<BranchProps> = ({
     >
       <Stack
         css={css({
```

Comment:
nitpick, can we try to get rid of the css from styled-system here?

Reply:
In this case it's using `paddingY: 11` which results in `44px`. We can update it to hard-coded `44px` and remove the `css` function, is that something we want?